### PR TITLE
Improves Start Work flows

### LINKED ISSUE #3832: Add integration connection button to title bar of select issue step
Put a plug icon in the title bar of the start work quick pick that takes them to connect integrations.

### LINKED ISSUE #3833: When no issues are returned, show an option to connect integrations
When users are in the start work flow and they don't see any issues when they get to the issue selection section, display an option in the menu that sends them to connect an integration.

### LINKED ISSUE #3836: Back button in "start work" goes to the wrong step

### PR DESCRIPTION
**Pull request by axosoft-ramint**
Closes #3832
Closes #3833
Closes #3836

- Adds title button to connect additional integrations.
- Adds telemetry event startWork/title/action
- Separates out "create branch" logic from Start Work and moves into its own telemetry even home/createBranch
- Removes all references to "start work type" including in telemetry
- Updates placeholder messaging
- Adds option to connect additional integrations or manage integrations when no issues are found.
- Adds telemetry event startWork/action
- Fixes back button and prevents refetching issues when clicked
- Updates freeze/resume logic into shared place and applies to all areas that use it
- Removes launchpad description item when at least one integration is connected in the connect step

### CODE REVIEW

File: src/constants.telemetry.ts [hunk 1]
```diff
@@ -230,6 +229,9 @@ export type TelemetryEvents = {
 		enabled: boolean;
 		version: string;
 	};
+	/** Temporary - sent when the user chooses to create a branch from the home view */
+	/** TODO: Replace this once we have source on all git commands*/
+	'home/createBranch': void;
```
Comment (eamodio) [inline src/constants.telemetry.ts:234]: I think it is good to keep this regardless of the git command plumbing

Comment (eamodio) [reply src/constants.telemetry.ts:234]: Maybe even add a `home/startWork' event to make it easier to track? @justinrobots

File: src/plus/launchpad/launchpad.ts [hunk 2]
```diff
@@ -978,20 +977,23 @@ export class LaunchpadCommand extends QuickCommand<State> {
...
-			createDirectiveQuickPickItem(Directive.Cancel, undefined, {
-				label: 'Launchpad prioritizes your pull requests to keep you focused and your team unblocked',
-				detail: 'Click to learn more about Launchpad',
-				iconPath: new ThemeIcon('rocket'),
-				onDidSelect: () =>
-					void executeCommand<OpenWalkthroughCommandArgs>(Commands.OpenWalkthrough, {
-						step: 'accelerate-pr-reviews',
-						source: 'launchpad',
-						detail: 'info',
+		const hasConnectedIntegration = some(context.connectedIntegrations.values(), c => c);
```
Comment (eamodio) [inline src/plus/launchpad/launchpad.ts:980]: Probably should filter to the supported integrations for Launchpad

**Review by eamodio — APPROVED** [review]
Couple of minor things can be in followup

File: src/plus/launchpad/launchpad.ts (hunk 2 above)
Comment (axosoft-ramint) [reply src/plus/launchpad/launchpad.ts:980]: `context.connectedIntegrations` is already filtered to supported Launchpad integrations where it is set up. See `getConnectedIntegrations` of `LaunchpadProvider`

File: src/constants.telemetry.ts (hunk 1 above)
Comment (axosoft-ramint) [reply src/constants.telemetry.ts]: I'll go ahead and add this. We can remove later if it feels redundant.

**Pull request merged**